core/gui/src/app/common/service/user/google-auth.service.ts

```diff
@@ -17,19 +17,30 @@ export class GoogleAuthService {
   public googleAuthInit(parent: HTMLElement | null) {
     this.http.get(`${AppSettings.getApiEndpoint()}/auth/google/clientid`, { responseType: "text" }).subscribe({
       next: response => {
-        window.onGoogleLibraryLoad = () => {
-          window.google.accounts.id.initialize({
-            client_id: response,
-            callback: (auth: CredentialResponse) => {
-              this._googleCredentialResponse.next(auth);
-            },
-          });
-          window.google.accounts.id.renderButton(parent, { width: 270 });
-          window.google.accounts.id.prompt();
+        const initializeGoogleLogin = () => {
```

Comment:
I feel this change is not correct. If this change is not required for this PR, let's have a offline discussion and fix google login button in a seperate PR.